### DISCUSSION TITLE
how can i use my  current terminal background theme ?

### DISCUSSION THREAD

**Discussion by ydv-sahitya** [helix-editor/helix#2567] [Q&A] [answered]
#### need help

I do not want to use helix theme. how can setup to terminal background theme ?

**Comment by the-mikedavis** [comment, marked as answer]
The base16 themes `base16_default_dark` `base16_terminal` `base16_default_light` and `base16_default` use whatever colors are configured by the terminal. You can try these with `:theme base16_default` and to set them permanently, you can add the `theme = "base16_default` key to the top of your `config.toml`: https://docs.helix-editor.com/configuration.html

**Comment by ydv-sahitya** [comment]
thank you, it's working.
